packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -47,6 +47,19 @@ describe("Hotkeys", () => {
         ).to.throw(HOTKEYS_HOTKEY_CHILDREN, "undefined");
     });
 
+    it("Decorator does not mutate the original class", () => {
+        class TestComponent extends React.Component<{}, {}> {
+            public renderHotkeys() {
+                return <Hotkeys />;
+            }
+        }
+
+        const TargettedTestComponent = HotkeysTarget(TestComponent);
+
+        // it's not the same Component
+        expect(TargettedTestComponent).to.not.equal(TestComponent);
```

Comment:
is this `expect...equal` the same as `assert.strictEqual`?

Reply:
Yeah equals is `===`